scripts/docs/preview_docc.sh

```diff
@@ -27,4 +26,4 @@
 ##
 ##===----------------------------------------------------------------------===##
 
-swift package --disable-sandbox preview-documentation --target $1
+xcrun swift package --disable-sandbox preview-documentation --target Tracing
```

Comment:
Well this doesn't seem that portable 😛

Reply:
What would be the alternative? Is there a portable way to pick up the currently selected toolchain? I don't think non-Darwin platforms even allow something like that.

Reply:
Platform checks to determine if it should be prefixed with `xcrun`

Reply:
Potentially, I'm not sure if the intention was to create a script that runs purely on macOS CI.